js/client.js

```diff
@@ -45,12 +53,13 @@ class Client {
 
   // Send a looking-for-game message.
   register() {
-    console.log("registering as " + this.name)
+    console.logVerbose("registering as " + this.name)
     this.send({op: "register", name: this.name})
   }
 
   forceUpdate() {
-    console.log("forceUpdate on the client was not overridden")
+    window.client.gameView.forceUpdate();
```

Comment:
The UI is never being updated after makeLocalMove without this call. I don't see anywhere forceUpdate is being called at all?